src/Microsoft.Tye.Hosting/Model/Application.cs

```diff
@@ -29,6 +29,9 @@ public Application(FileInfo source, Dictionary<string, Service> services)
 
         public string? Network { get; set; }
 
+        // All services and application run on the container host.
+        public bool UseHostNetwork { get; set; }
```

Comment:
I think you need to parse this in config, right? Do you intend for this to be part of tye.yaml and/or a command line arg to tye run?

Reply:
Ah nvrm, you use podman existing as that check.

Little bit confusing, should this variable be called IsPodman for now?